mentoring/tests/integration/test_dashboard.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2014-2015 Harvard, edX & OpenCraft
```

Comment:
Not sure why this is weird? It's the official recommendation from the FSF on how to apply the license (cf https://www.gnu.org/licenses/agpl-3.0.html ), and a lot of project do this. It can be fine to only have it in one file when there is a single copyright holder, like for most edX repositories, but since we don't do copyright assignment, it's usual to have individual copyright claims on individual files. Especially because the authors can be the same for different coypright holders, so I don't see another way to be explicit about who has ownership on which part, no?

Now on the current file for example, the copyright is fully Harvard - they are paying for the current feature, edX and OpenCraft have no claims on it.